src/Api/Data/DbTask.cs

```diff
@@ -6,19 +6,28 @@
 using Microsoft.Extensions.DependencyInjection;
 using Api.Models.Entity;
 using System;
+using Microsoft.AspNetCore.Hosting;
 
 namespace Api.Data
 {
     public static class DbTask
```

Comment:
In previous PR: The issue was, the migrations did not apply to anyother environments but this check will make the migrations apply to other environments except test that is the framework environment. I tried my maximum to work on this. Faced a lot issues and finally came up with this.